### PR TITLE
Add parameter user_shell

### PR DESCRIPTION
**Pull request by dhoppe**
This pull request depends on #314

### CODE REVIEW

**Comment by dhoppe** [comment]
We should refactor the acceptance tests in a separate pull request.

**Comment by bastelfreak** [comment]
removed the `enhancement` label because the changelog generator might be confused  with two labels for different sections.

File: manifests/broker.pp [hunk 1]
```diff
@@ -121,6 +124,7 @@
   Optional[String[1]] $package_name          = $kafka::params::package_name,
   String[1] $package_ensure                  = $kafka::params::package_ensure,
   String[1] $user_name                       = $kafka::params::user_name,
+  String[1] $user_shell                      = $kafka::params::user_shell,
```
Comment (bastelfreak) [inline manifests/broker.pp:127]: Isn't this always an absolute path?
```suggestion
  Stdlib::Absolutepath $user_shell           = $kafka::params::user_shell,
```

File: manifests/consumer.pp [hunk 2]
```diff
@@ -109,6 +112,7 @@
   Optional[String[1]] $package_name          = $kafka::params::package_name,
   String[1] $package_ensure                  = $kafka::params::package_ensure,
   String[1] $user_name                       = $kafka::params::user_name,
+  String[1] $user_shell                      = $kafka::params::user_shell,
```
Comment (bastelfreak) [inline manifests/consumer.pp:115]:
```suggestion
  Stdlib::Absolutepath $user_shell           = $kafka::params::user_shell,
```

File: manifests/broker.pp (hunk 1 above)
Comment (dhoppe) [reply manifests/broker.pp:127]: Thx

File: manifests/init.pp [hunk 3]
```diff
@@ -91,6 +94,7 @@
   Optional[String[1]] $proxy_type     = $kafka::params::proxy_type,
   String[1] $package_ensure           = $kafka::params::package_ensure,
   String[1] $user_name                = $kafka::params::user_name,
+  String[1] $user_shell               = $kafka::params::user_shell,
```
Comment (bastelfreak) [inline manifests/init.pp:97]:
```suggestion
  Stdlib::Absolutepath $user_shell    = $kafka::params::user_shell,
```

File: manifests/producer.pp [hunk 4]
```diff
@@ -113,6 +116,7 @@
   Optional[String[1]] $package_name          = $kafka::params::package_name,
   String[1] $package_ensure                  = $kafka::params::package_ensure,
   String[1] $user_name                       = $kafka::params::user_name,
+  String[1] $user_shell                      = $kafka::params::user_shell,
```
Comment (bastelfreak) [inline manifests/producer.pp:119]:
```suggestion
  Stdlib::Absolutepath $user_shell           = $kafka::params::user_shell,
```

**Pull request merged**